src/components/orders/DetailsTable/index.tsx

```diff
@@ -83,13 +83,15 @@ const tooltip = {
   expiration:
     'The date and time at which an order will expire and effectively be cancelled. Depending on the type of order, it may have partial fills upon expiration.',
   type: 'An order can be either a Buy or Sell order. In addition, an order may be of type "Fill or Kill" (no partial fills) or a regular order (partial fills allowed).',
-  amount: 'The total sell and buy amount for this order.',
+  amount: 'The total sell and buy amount for this order. Sell amount includes the fee.',
   priceLimit:
-    'The limit price is the price at which this order shall be (partially) filled, in combination with the specified slippage.',
-  priceExecution: 'The actual price at which this order has been matched and executed.',
+    'The limit price is the price at which this order shall be (partially) filled, in combination with the specified slippage. The fee is already deduced from the sell amount',
+  priceExecution:
+    'The actual price at which this order has been matched and executed, after deducting the fees from the sold amount.',
```

Comment:
Are we sure we want to specify this? It might confuse more than help. Like "after deducting the fees" might mean that the price is calculated on the amounts after deducting the fees (so looks better), but can also mean that the price includes fees (so it looks worse)

Probably is dine as we had the tooltip? I would ask @MindyCoW